signac/contrib/filterparse.py

```diff
@@ -84,6 +191,22 @@ def _parse_simple(key, value=None):
 
 
 def parse_filter_arg(args, file=sys.stderr):
+    """
+
+    Parameters
+    ----------
+    args :
+
```

Comment:
Not sure how to document the type here. It's supposed to be a sequence (e.g. list or tuple) in my recollection.
```suggestion
    args : sequence of str
        Filter arguments to parse.
```